src/components/pagination/demo/mock-pagination-data.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * These functions mimic the structure of WordPress's pagination objects.
+ *
+ * @see https://developer.wordpress.org/reference/functions/paginate_links/
+ * @see https://timber.github.io/docs/guides/pagination/
+ */
+
+function mockPageData(page = 1, current = 1) {
+  return {
+    title: page,
+    text: page,
+    name: page,
+    current: page === current,
+    link: page !== current && `#page-${page}`,
+  };
+}
+
+export default function mockPaginationData({
+  current = 1,
+  midSize = 2,
+  total = 36,
+} = {}) {
```

Comment:
I'm not familiar with this options syntax. Are you passing an options object and then setting the default to an empty object?

Reply:
See: https://simonsmith.io/destructuring-objects-as-function-parameters-in-es6#allowing-the-configuration-object-to-be-optional

Reply:
Oh cool! #TIL

Reply:
- [x] Added `@param` comments to make structure more clear